### PR TITLE
Migrate some classes to Kotlin

### PR DESCRIPTION
**Pull request by ferranpons**
Depends on the PR #167

### CODE REVIEW

**Comment by CristianGM** [comment]
Are you going to convert a lot of files I to a single PR?

**Comment by ferranpons** [comment]
@CristianGM I will convert only the less problematic files. Only interfaces and "not key" classes.

**Comment by CristianGM** [comment]
IMHO convert just one on the PR to prove it works, don't do a big PR converting all now

**Comment by ferranpons** [comment]
Ping @gerardpedreny @marcserrascmspain 😅

File: leku/src/main/java/com/schibstedspain/leku/geocoder/api/AddressBuilder.kt [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  }
+
+  @Throws(JSONException::class)
+  private fun parseAddress(jsonObject: JSONObject): Address {
+    val location = jsonObject.getJSONObject("geometry").getJSONObject("location")
+    val latitude = location.getDouble("lat")
+    val longitude = location.getDouble("lng")
+
+    val components = getAddressComponents(jsonObject.getJSONArray("address_components"))
+
```
Comment (marcserrascmspain) [inline leku/src/main/java/com/schibstedspain/leku/geocoder/api/AddressBuilder.kt]: empty lines

**Pull request merged**